src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -916,10 +856,7 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
   const renderRadioToggle = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Enable venue radio</h4>
-      <label id="showRadio" className="switch">
-        <input type="checkbox" id="showRadio" name="showRadio" ref={register} />
-        <span className="slider round" />
-      </label>
+      <Toggler id="showRadio" forwardedRef={register} />
```

Comment:
There's no need for an `id`, because `Checkbox` nests it's `input` within the `label`, so they are inherently connected without needing the `id`.

```suggestion
      <Toggler name="showRadio" forwardedRef={register} />
```